PWGHF/D2H/Tasks/taskLcToK0sP.cxx

```diff
@@ -39,14 +39,14 @@ struct HfTaskLcToK0sP {
   HistogramRegistry registry{
     "registry",
     {// data
-     {"hMass", "cascade candidates;inv. mass (#pi K) (GeV/#it{c}^{2});entries", {HistType::kTH1F, {{500, 0.0f, 5.0f}}}},
+     {"hMass", "cascade candidates;inv. mass (p K_{S}^{0}) (GeV/#it{c}^{2});entries", {HistType::kTH2F, {{500, 1.5f, 3.0f}, {30, 0.0f, 30.0f, "#it{p}_{T} (GeV/#it{c})"}}}},
      {"hPtCand", "cascade candidates;candidate #it{p}_{T} (GeV/#it{c});entries", {HistType::kTH1F, {{100, 0.0f, 10.0f}}}},
-     {"hPtBach", "cascade candidates;bachelor #it{p}_{T} (GeV/#it{c});entries", {HistType::kTH1F, {{100, 0.0f, 10.0f}}}},
-     {"hPtV0", "cascade candidates;v0 #it{p}_{T} (GeV/#it{c});entries", {HistType::kTH1F, {{100, 0.0f, 10.0f}}}},
-     {"hd0Bach", "cascade candidates;bachelor DCAxy to prim. vertex (cm);entries", {HistType::kTH1F, {{100, -1.0f, 1.0f}}}},
-     {"hd0V0pos", "cascade candidates;pos daugh v0 DCAxy to prim. vertex (cm);entries", {HistType::kTH1F, {{100, -5.0f, 5.0f}}}},
-     {"hd0V0neg", "cascade candidates;neg daugh v0 DCAxy to prim. vertex (cm);entries", {HistType::kTH1F, {{100, -5.0f, 5.0f}}}},
-     {"hV0CPA", "cascade candidates;v0 cosine of pointing angle;entries", {HistType::kTH1F, {{110, -0.98f, 1.1f}}}},
+     {"hPtBach", "cascade candidates;bachelor #it{p}_{T} (GeV/#it{c});entries", {HistType::kTH2F, {{100, 0.0f, 10.0f}, {30, 0.0f, 30.0f, "#it{p}_{T} (GeV/#it{c})"}}}},
+     {"hPtV0", "cascade candidates;v0 #it{p}_{T} (GeV/#it{c});entries", {HistType::kTH2F, {{100, 0.0f, 10.0f}, {30, 0.0f, 30.0f, "#it{p}_{T} (GeV/#it{c})"}}}},
+     {"hd0Bach", "cascade candidates;bachelor DCAxy to prim. vertex (cm);entries", {HistType::kTH2F, {{100, -1.0f, 1.0f}, {30, 0.0f, 30.0f, "#it{p}_{T} (GeV/#it{c})"}}}},
+     {"hd0V0pos", "cascade candidates;pos daugh v0 DCAxy to prim. vertex (cm);entries", {HistType::kTH2F, {{200, -5.0f, 5.0f}, {30, 0.0f, 30.0f, "#it{p}_{T} (GeV/#it{c})"}}}},
+     {"hd0V0neg", "cascade candidates;neg daugh v0 DCAxy to prim. vertex (cm);entries", {HistType::kTH2F, {{200, -5.0f, 5.0f}, {30, 0.0f, 30.0f, "#it{p}_{T} (GeV/#it{c})"}}}},
+     {"hV0CPA", "cascade candidates;v0 cosine of pointing angle;entries", {HistType::kTH2F, {{100, 0.98f, 1.0001f}, {30, 0.0f, 30.0f, "#it{p}_{T} (GeV/#it{c})"}}}},
```

Comment:
- Same comment as above on `entries` and the pT binning for these six histograms
- Unclear why you didn't make `hEta` a 2D histogram

Reply:
OK, done.